Clarifications on private location scaling (#9138)
DataDog/documentation: content/en/synthetics/private_locations/_index.md

You can easily **horizontally scale** your private locations by adding or removing workers to it. You can run several containers for one private location with one single configuration file. Each worker would then request `N` tests to run depending on its number of free slots: when worker 1 is processing tests, worker 2 requests the following tests, etc.

> You can also **vertically scale** your private locations using the [`concurrency` parameter][11] value to adjust the number of available slots on your private locations (ie. number of tests your private location workers can run in parallel).

Reviewer: ```suggestion
You can also **vertically scale** your private locations using the [`concurrency` parameter][11] to adjust the number of available slots on your private locations, for example: the number of tests your private location workers can run in parallel.
```

> You can also **vertically scale** your private locations using the [`concurrency` parameter][11] to adjust the number of available slots on your private locations, for example: the number of tests your private location workers can run in parallel.

Author: ```suggestion
You can also **vertically scale** your private locations using the [`concurrency` parameter][11] to adjust the number of available slots on your private locations. These slots are the number of tests your private location workers can run in parallel.
```
